alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/cognitiveagents/characteristics/individual/Gender.kt

```diff
@@ -3,5 +3,16 @@ package it.unibo.alchemist.model.cognitiveagents.characteristics.individual
 enum class Gender : IndividualCharacteristic {
 
     MALE,
-    FEMALE
+    FEMALE;
+
+    companion object {
+        private val MALE_KEYWORDS = setOf("male", "m", "MALE", "M")
+        private val FEMALE_KEYWORDS = setOf("female", "f", "FEMALE", "F")
+
+        fun getCategory(gender: String): Gender = when {
```

Comment:
prefer `fromString`, or `fromSomething`. You don't get anything, it's very Java-ish. Be idiomatic.